src/versioning/versioning.py

```diff
@@ -691,7 +691,9 @@ def main(argv=None) -> int: # IGNORE:C0111
     program_version = "v%s" % __version__
     program_build_date = str(__updated__)
     program_version_message = '%%(prog)s %s (%s)' % (program_version, program_build_date)
-    program_shortdesc = __import__('__main__').__doc__.split("\n")[1]
+    #program_shortdesc = __import__('__main__').__doc__.split("\n")[1]
```

Comment:
@ucbjrl I'm not sure what this is doing, my hack is just to get it to run right now.